packages/frontend/src/pages/scaling-projects/view/ProjectPage.tsx

```diff
@@ -1,39 +1,51 @@
 import React from 'react'
 
 import {
-  Chart,
-  ChartProps,
   Footer,
   FooterProps,
   HeaderProps,
   Navbar,
   NavbarProps,
 } from '../../../components'
 import { PageContent } from '../../../components/PageContent'
+import { DesktopProjectNavigation } from '../../../components/project/navigation/DesktopProjectNavigation'
+import { MobileProjectNavigation } from '../../../components/project/navigation/MobileProjectNavigation'
+import { ProjectDetailsSection } from '../props/getProjectDetails'
 import { ProjectDetails, ProjectDetailsProps } from './ProjectDetails'
 import { ProjectHeader, ProjectHeaderProps } from './ProjectHeader'
 
 export interface ProjectPageProps {
   navbar: NavbarProps
   header: HeaderProps
   projectHeader: ProjectHeaderProps
-  chart: ChartProps
   projectDetails: ProjectDetailsProps
   footer: FooterProps
 }
 
 export function ProjectPage(props: ProjectPageProps) {
+  const sections = props.projectDetails.items.filter(
+    (i): i is ProjectDetailsSection => !i.excludeFromNavigation,
+  )
   return (
     <>
       <Navbar {...props.navbar} />
+      <div className="sticky top-0 z-[1000] md:hidden">
+        <MobileProjectNavigation sections={sections} />
+      </div>
       <PageContent mobileFull>
         <ProjectHeader {...props.projectHeader} />
-        <Chart
-          {...props.chart}
-          mobileFull
-          isUpcoming={props.projectDetails.isUpcoming}
-        />
-        <ProjectDetails {...props.projectDetails} />
+        <div className="mt-8 grid grid-cols-7 gap-x-12">
+          <div className="col-span-2 hidden md:block">
```

Comment:
This div could be part of DesktopProjectNavigation

Reply:
Im not sure if it's good, you are referring to the grid-cols class of parent div, if you put it inside DesktopProjectNavigation it will be unclear I believe